apps/smart-contracts/core/scripts/ArbitragePools.ts

```diff
@@ -32,6 +32,12 @@ export type PoolPrices = {
   shortPoolPrice: BigNumber
 }
 
+export enum ArbitrageType {
```

Comment:
nit:
```suggestion
export enum ArbitrageStrategy {
```
to be consistent with function name